packer/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+pushd `dirname "$0"`
+
+if [ -e manifest.json ]; then
```

Comment:
If `packer.json` changes, will Packer still run correctly? Now that there is an `update_git` script, where do we delete `manifest.json` to make sure Packer runs on changes??